lib/internal/navigator.js

```diff
@@ -24,6 +30,8 @@ class Navigator {
   // Private properties are used to avoid brand validations.
   #availableParallelism;
   #userAgent = `Node.js/${nodeVersion.slice(1, nodeVersion.indexOf('.'))}`;
+  #language = Intl?.Collator().resolvedOptions().locale || 'en-US';
```

Comment:
are there primordials for Intl as well?

Reply:
I did 
console.log('Intl' in primordials)
and get false as output.

Can I resolve this?

Reply:
Yes, per https://github.com/nodejs/TSC/issues/1439 we're not requiring primordials everywhere anymore.

Reply:
That change hasn't been made yet, so they're still required everywhere.

However, if there's no Intl primordials then obviously there's nothing to do here, and it's fine to resolve it.